sdk/storage/azblob/sas/query_params.go

```diff
@@ -23,7 +23,7 @@ const (
 
 var (
 	// Version is the default version encoded in the SAS token.
-	Version = "2019-12-12"
+	Version = "2020-02-10"
```

Comment:
This value is also used in the `SignWithSharedKey()` APIs. You're sure this won't break them?

Reply:
I'm confident! The string-to-sign version 2020-12-06 is the one that will need some changes, but 2020-02-10 will fall under this format (https://learn.microsoft.com/en-us/rest/api/storageservices/create-service-sas#version-2018-11-09-and-later) which is our current string-to-sign behavior.

Reply:
Thanks for confirming!